src/Spectre.libGenetic/Generation.cpp

```diff
@@ -44,6 +44,19 @@ Generation::Generation(std::vector<Individual> &&generation):
     else { }
 }
 
+
+Generation::Generation(size_t size, size_t individualSize, size_t initialFillup, Seed seed)
```

Comment:
`Generation` should be *data container*, not an *algorithm*. This part should be extracted into a `RandomGenerationFactory` which main purpose will be construction of random generation, as discussed.